mne/time_frequency/tfr.py

```diff
@@ -1392,7 +1421,6 @@ def __setstate__(self, state):
 
         defaults = dict(
             method="unknown",
-            dims=("epoch", "channel", "freq", "time")[-state["data"].ndim :],
```

Comment:
Have removed `dims` being set in `BaseTFR` since the possibility of the optional `epoch` and `taper` dimensions makes it really difficult to disentangle here. It's much easier to handle this in the individual `RawTFR`, `EpochsTFR`, and `AverageTFR` classes.